core/src/main/scala/com.softwaremill.kmq/redelivery/MarkersQueue.scala

```diff
@@ -1,46 +1,37 @@
 package com.softwaremill.kmq.redelivery
```

Comment:
I think we should keep a single implementationof the redeliverer, so now that the new one works the old one can be removed

Reply:
I was going a general cleanup of previous implementation after the new one is ready. The cleanup would include removing no-more-used classes and updating docs. Mayby this could be done in a separate PR for branch `release-0.3`?